src/main/python/extraction/sample_extract.py

```diff
@@ -0,0 +1,58 @@
+from image_processing.build_director import BuildDirector
+from extraction.text_manager import TextManager
+from extraction.simplification_manager import SimplificationManager
+from extraction.barcode_manager import BarCodeManager
+import pytesseract
+from PIL import Image
+import cv2
+import os
+
+DESKTOP = os.path.join(os.path.join(os.path.expanduser('~')), 'Desktop')
+
+
+class TextExtractor:
+    """
+        The TextExtractor extracts text from the ID image
+    """
+    def extract(self, img):
+        """
+        This function is a sample that demonstrates how text would be extracted
+        Author(s):
+            Nicolai van Niekerk
+        Args:
+            image: The image of the ID that contains the text tom be extracted
+        Returns:
+            id_details: JSON obj (The extracted information)
+        Todo:
```

Comment:
Why is there an empty Todo tag? If there is nothing to do, don't add the tag, and if there is something to do, fill in the correct details :wink: